### PR TITLE
frontend updates

### PR DESCRIPTION
**Pull request by ATHARVA262005**

### CODE REVIEW

File: src/PageTemplate/DepartmentTemplate.jsx [hunk 1]
```diff
@@ -47,9 +44,6 @@ const DepartmentTemplate = ({ cardsData, teachers, title, description, events, g
         <div className="bg-gray-100 mx-10" id='eventSection'>
           <EventDepartmentSection events={events} />
         </div>
-        <div className="bg-gray-100 mx-10" id='gallerySection'>
-          <GallerySection galleryImages={galleryImages} />
-        </div>
```
Comment (SAYUK09) [inline src/PageTemplate/DepartmentTemplate.jsx:52]: Don't Delete. Only Comment.

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Take a look at the requested changes

File: src/Components/HeroVid/HeroVid.css [hunk 2]
```diff
@@ -2,6 +2,7 @@
     position: relative;
     height: 100vh;
     overflow: hidden;
+    z-index: 0;;
   }
```
Comment (SAYUK09) [inline src/Components/HeroVid/HeroVid.css:6]: double semi colon

**Review by SAYUK09 — CHANGES_REQUESTED** [review]

File: src/Components/HeroVid/HeroVid.css (hunk 2 above)
Comment (ATHARVA262005) [reply src/Components/HeroVid/HeroVid.css:6]: herovid.css is unused , we use tailwind css now its just for backup

File: src/PageTemplate/DepartmentTemplate.jsx (hunk 1 above)
Comment (ATHARVA262005) [reply src/PageTemplate/DepartmentTemplate.jsx:52]: that section isn't necessary now

File: src/Components/HeroVid/HeroVid.css (hunk 2 above)
Comment (SAYUK09) [reply src/Components/HeroVid/HeroVid.css:6]: If it's not being used, can't we delete it?
And if we want to keep it, can we please get rid of the syntax error.

File: src/PageTemplate/DepartmentTemplate.jsx (hunk 1 above)
Comment (SAYUK09) [reply src/PageTemplate/DepartmentTemplate.jsx:52]: But won't we need it in future?
It would be easy to just uncomment and start using it, won't it?

**Pull request merged**